### DISCUSSION TITLE
Should we remove the changelog file?

### DISCUSSION THREAD

**Discussion by git-hulk** [apache/kvrocks#731] [Polls]
I'm wondering if we can remove the `Changelog` file after PR https://github.com/apache/incubator-kvrocks/pull/729. The main point is it will be hard to maintain since we need to call a vote before drafting the release.

**Comment by tisonkun** [comment]
I agree that we don't have to maintain the ChangeLog file in our main repo. However, it will be useful to write a release note and publish on our official website. See also [Apache ZooKeeper's Download page](https://zookeeper.apache.org/releases.html) as a simple solution.

**Comment by PragmaTwice** [comment]
+1, we can move these changelog out of the repo, to like github release page or website

**Comment by git-hulk** [comment]
Thanks all, I will delete this changelog and maintain it on our website. cc @ShooterIT

**Comment by ShooterIT** [comment]
I prefer to keep a history or all version descriptions file in the repo, after getting the repo code, users can see all things about it.

**Reply by git-hulk** [reply]
Yes, it's a good idea to let users see or search the history in one place,
but our changelog in the unstable branch is out of date, and NOT easy to keep fresh.
So I think we can maintain it on our website instead of the repo.
